temporal/fx.go

```diff
@@ -865,18 +866,20 @@ type SpanExporterInputs struct {
 	Config     *config.Config `optional:"true"`
 }
 
+var tracingReady atomic.Bool
```

Comment:
I tried a few things and settled on this approach.

I tried registering it OnStart, but then the logs are still written using the default logger.
I tried de-registering it on OnStop, but then it was still happening at startup.

Reply:
Global var in almost root module :man_facepalming:. At least it is not exported.

Reply:
Oh, your comment made me realize that our functional tests actually run multiple fx instances!
Is there an fx context or sth I can attach it to?